src/components/utils/DrawerComment/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import { Global } from "@emotion/react";
+import { styled } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+import { grey } from "@mui/material/colors";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import Skeleton from "@mui/material/Skeleton";
+import Typography from "@mui/material/Typography";
+import SwipeableDrawer from "@mui/material/SwipeableDrawer";
+import { Puller, Root, StyledBox, Title } from "./styles";
+
+const drawerBleeding = 56;
+
+interface Props {
+  window?: () => Window;
+  opened: boolean;
+  title?: string;
+  children: React.ReactNode;
+}
+
+const SwipeableEdgeDrawer = ({ opened, children, title }: Props) => {
+  const [open, setOpen] = React.useState(false);
+
+  const toggleDrawer = (newOpen: boolean) => () => {
+    setOpen(newOpen);
+  };
+  React.useEffect(() => {
```

Comment:
lets try to keep using the same pattern, importing directly useState and useEffect {} from react, instead of * as React and use React.useState...